gqltesting/testing.go

```diff
@@ -18,6 +18,7 @@ type Test struct {
 	OperationName  string
 	Variables      map[string]interface{}
 	ExpectedResult string
+	ErrorExpected  bool
```

Comment:
What if we provided `Errors []*errors.QueryError` to allow for checking that the errors received were all expected?

This would let us do something like:
```go
func checkErrors(t *testing.T, expected, actual []*errors.QueryError) {
	expectedCount, actualCount := len(expected), len(actual)
	if expectedCount > 0 {
		t.Fatal("TODO: compare expected errors with actual errors")
	}

	for _, err := range actual {
		t.Errorf("unexpected error: '%s'", err)
	}

	if actualCount > 0 {
		t.Fatal("got unexpected errors")
	}
}
```

Reply:
I dig that.

Reply:
Also, thanks for the feedback, I really do appreciate it!